internal/service/lightsail/container_service_test.go

```diff
@@ -0,0 +1,1306 @@
+package lightsail_test
+
+import (
```

Comment:
Thorough AccTest support for creation and deletion of resources. Really nice work!

What we would like to see is support for modification of those resources to assert that the state is being imported correctly and resources are appropriately updated in place. For example, changing the image tag of an existing deployment to verify the new image is proper loaded.

See example here for importing state and modifying config, and then checking the values to assert the changes happened.

- https://github.com/hashicorp/terraform-provider-aws/blob/c942e1dbacc12e4f53ae4ab61bdf1fa0e417f5a2/internal/service/lexmodels/bot_test.go#L128
- https://github.com/hashicorp/terraform-provider-aws/blob/c942e1dbacc12e4f53ae4ab61bdf1fa0e417f5a2/internal/service/lexmodels/bot_test.go#L245